_posts/0000-01-02-MEDIdesign.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
_In the bleak mid-winter_ sang Uncle Theodore gaily as he stared out from the battlements of his castle in the late evening sun.